--- gnome-shell-3.26.0/meson.build.orig	2017-09-12 17:26:59.000000000 +0200
+++ gnome-shell-3.26.0/meson.build	2017-10-01 20:25:31.854788015 +0200
@@ -52,7 +52,7 @@
 desktopdir = join_paths(datadir, 'applications')
 ifacedir = join_paths(datadir, 'dbus-1', 'interfaces')
 localedir = join_paths(datadir, 'locale')
-mozplugindir = join_paths(libdir, 'mozilla', 'plugins')
+mozplugindir = join_paths(libdir, 'browser-plugins')
 portaldir = join_paths(datadir, 'xdg-desktop-portal', 'portals')
 schemadir = join_paths(datadir, 'glib-2.0', 'schemas')
 servicedir = join_paths(datadir, 'dbus-1', 'services')
