--- gnome-shell-3.38.0/docs/reference/shell/shell-docs.sgml.orig	2020-09-14 22:48:39.373818900 +0200
+++ gnome-shell-3.38.0/docs/reference/shell/shell-docs.sgml	2020-09-22 20:00:18.085865408 +0200
@@ -37,10 +37,6 @@
     <xi:include href="xml/shell-tray-manager.xml"/>
   </chapter>
   <chapter>
-    <title>Recorder</title>
-    <xi:include href="xml/shell-recorder.xml"/>
-  </chapter>
-  <chapter>
     <title>Integration helpers and utilities</title>
     <xi:include href="doc-gen-org.gnome.Shell.SearchProvider.xml"/>
     <xi:include href="doc-gen-org.gnome.Shell.SearchProvider2.xml"/>
