--- gnome-shell-3.10.0.1/src/Makefile.am.wiget	2013-09-29 15:48:58.693327696 +0200
+++ gnome-shell-3.10.0.1/src/Makefile.am	2013-09-29 15:52:08.644347011 +0200
@@ -301,7 +301,7 @@ libgnome_shell_base_la_LIBADD = $(libgno
 libgnome_shell_base_la_CPPFLAGS = $(gnome_shell_cflags)
 
 libgnome_shell_la_LDFLAGS = $(libgnome_shell_ldflags)
-libgnome_shell_la_LIBADD = $(GNOME_SHELL_LIBS) $(MUTTER_LIBS) libgnome-shell-base.la
+libgnome_shell_la_LIBADD = $(GNOME_SHELL_LIBS) $(MUTTER_LIBS) $(BLUETOOTH_LIBS) libgnome-shell-base.la
 libgnome_shell_la_CPPFLAGS = $(MUTTER_CFLAGS) $(gnome_shell_cflags)
 
 if HAVE_MUTTER_WAYLAND
